reference-architectures/cloud_deploy_flow/CloudBuild/buildCloudRun.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  dir: 'CloudRun/'
+  args: ['build', '-f', 'dockerfile', '-t', 'us-central1-docker.pkg.dev/$PROJECT_ID/random-date-app/app:latest', '.']
+- name: 'gcr.io/cloud-builders/docker'
+  dir: 'CloudRun/'
+  args: ['push', 'us-central1-docker.pkg.dev/$PROJECT_ID/random-date-app/app:latest']
+- name: 'gcr.io/cloud-builders/gcloud'
+  dir: 'CloudRun/'
+  waitFor: ['-']
+  entrypoint: 'bash'
+  args:
+  - '-c'
```

Comment:
Consider setting `-o errexit`, `-o nounset`, and `-o pipefail` to make this command a bit more robust.